packages/lesswrong/components/spotlights/SpotlightFeedItem.tsx

```diff
@@ -0,0 +1,365 @@
+      marginBottom: 8,
+    },
+    subtitle: {
+      ...theme.typography.postStyle,
+      ...theme.typography.italic,
+      color: theme.palette.grey[700],
+      fontSize: "1.3rem",
+      marginTop: -1,
+    },
+    imageContainer: {
+      margin: "-30px -16px -20px -16px",
+      position: "relative",
+      zIndex: 1,
+      alignSelf: "stretch",
+      display: "flex",
+      justifyContent: "flex-end",
+    },
+    image: {
+      maxWidth: "100%",
+      height: "auto",
+      objectFit: "cover",
+      objectPosition: "right center",
+      maxHeight: 250,
+      borderRadius: `${theme.borderRadius.default}px 0 0 ${theme.borderRadius.default}px`,
+    },
+    imageDualFade: buildFadeMask([
+      "transparent 5%",
+      `${theme.palette.text.alwaysWhite} 25%`,
+      `${theme.palette.text.alwaysWhite} 65%`,
+      "rgba(255,255,255,0.5) 80%",
+      "transparent 100%"
+    ]),
+    imageFade: buildFadeMask([
+      "transparent 0",
+      `${theme.palette.text.alwaysWhite} 80%`,
+      `${theme.palette.text.alwaysWhite} 100%`,
+    ]),
+    imageFadeCustom: buildFadeMask([
+      "transparent 0",
+      "transparent 30%",
+      `${theme.palette.text.alwaysWhite} 90%`,
+      `${theme.palette.text.alwaysWhite} 100%`,
+    ]),
+    descriptionArea: {
+      marginTop: 5,
+      zIndex: 3,
+      display: "flex",
+      flexDirection: "column",
+      gap: 8,
+    },
+    description: {
+      ...descriptionStyles(theme),
+      fontSize: "1.3rem",
+      opacity: 0.9,
+    },
+    postPadding: {
+      paddingBottom: 12
+    },
+    author: {
+      // marginTop: 4,
+      color: theme.palette.grey[600],
+      marginBottom: 4,
+    },
+    authorName: {
+      color: theme.palette.primary.main,
+    },
+    draftButton: {
+      [theme.breakpoints.up('md')]: {
+        position: "absolute",
+        top: 35,
+        right: -28,
+      },
+      [theme.breakpoints.down('sm')]: {
+        position: "absolute",
+        top: 33,
+        right: 8
+      },
+    },
+    deleteButton: {
+      [theme.breakpoints.up('md')]: {
+        position: "absolute",
+        bottom: 0,
+        right: -28,
+      },
+      [theme.breakpoints.down('sm')]: {
+        position: "absolute",
+        bottom: 0,
+        right: 8
+      },
+    },
+    editAllButtonIcon: {
+      width: 20
+    },
+    metaData: {
+      textAlign: "right",
+      paddingTop: 6,
+      paddingBottom: 12
+    },
+    splashImage: {
+      filter: "brightness(1.2)",
+    },
+    splashImageContainer: {
+      position: "absolute",
+      top: 0,
+      right: 0,
+      width: "100%",
+      height: "100%",
+      overflow: "hidden",
+      [theme.breakpoints.down('xs')]: {
+        height: "100% !important",
+      }
+    },
+    reverseIcon: {
+      transform: "rotate(180deg)",
+    },
+    imageContainerWithAuthor: {
+      marginTop: -30,
+    },
+  }),
+  { stylePriority: -1 }
+);
+
+function getSpotlightDisplayTitle(spotlight: SpotlightDisplay): string {
```

Comment:
Is this just copied directly from `SpotlightItem`?

Reply:
I've deduplicated some by exporting from there